dockerfiles/Dockerfile

```diff
@@ -2,20 +2,20 @@
 FROM golang:1.10-alpine3.7
 MAINTAINER David Chang <dchang@linkernetworks.com>
 # TODO fix path after move out
-WORKDIR /go/src/bitbucket.org/linkernetworks/vortex
+WORKDIR /go/src/github.com/linkernetworks/vortex
 
 RUN apk add --no-cache protobuf ca-certificates git
 
 COPY config /cv-tracker/config
 # TODO fix path after move out
-COPY . /go/src/bitbucket.org/linkernetworks/vortex/src
-COPY Makefile /go/src/bitbucket.org/linkernetworks/vortex
+COPY . /go/src/github.com/linkernetworks/vortex/src
+COPY Makefile /go/src/github.com/linkernetworks/vortex
 
 ENV CGO_ENABLED 0
 ENV GOOS linux
 ENV TIMEZONE "Asia/Shanghai"
 ARG BUILD_MODE=release
-ARG BRAND=aurora
+ARG BRAND=vortex
```

Comment:
@hwchiu BRAND should be removed not changed?

Reply:
This BRAND is for L24.

Reply:
We don't need the BRAND here, that's for our previous project.

Reply:
Also modify the L24 to remove the BRAND, Thanks